gemreadout/data/slot_table.csv

```diff
@@ -1 +1 @@
-slot_table_904_may23.csv
```

Comment:
Rather than having the slot table be point to some fixed file in the release tree, let's make it a link to a central place.
Especially since there is absolutely no reason that different users would need a different file.